ios/Podfile.lock

```diff
@@ -1,126 +0,0 @@
-PODS:
```

Comment:
Why are we deleting the lock file? Shouldn't we update it?

Is it because we are not loading pods from `node_modules`?

Reply:
The lockfile was deleted on the diff, I also didn't understand why.